httpbp/prometheus.go

```diff
@@ -16,6 +16,22 @@ const (
 	endpointLabel   = "http_endpoint"
 )
 
+var payloadSizeBuckets = []float64{
```

Comment:
You missed the comment about adding a comment :)

Reply:
Ah man, I did.

Reply:
Take 3.